src/PrestaShopBundle/Form/Admin/Sell/Product/Options/OptionsType.php

```diff
@@ -42,6 +43,14 @@ class OptionsType extends TranslatorAwareType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
+            ->add('published_date', DatePickerType::class, [
+                'required' => false,
+                'label' => $this->trans('Date of publication', 'Admin.Catalog.Feature'),
+                'label_subtitle' => $this->trans('When do you want your product to appear?', 'Admin.Catalog.Feature'),
```

Comment:
@matthieu-rolland this suggests it's a feature to plan the publication of the product

Reply:
true, I knew this was bad, I copy pasted `Where do you want your product to appear?` and replaced `appear` with `published`, I know it's wrong, I'm just waiting for proper label with @MatShir 

that's why this PR is in draft, I just opened it to check the CI...